test/test.py

```diff
@@ -367,31 +266,27 @@ def test_kinetics(mechname, fuel, stoich_ratio, dt, usr_np):
         print("err_omega = ", err_omega)
         print("err_r = ", err_r)
         print()
-
         # Compare
         assert err_r < 1.0e-10
         assert err_omega < 1.0e-10
 
-    return
 
+@pytest.mark.skipif(jnp is None, reason="JAX not installed")
+def test_autodiff_accuracy(request):
+    backend = BACKENDS[request.config.getoption("backend").lower()]
+    if backend != PythonBackend:
+        pytest.skip("JAX only supported with Python backend")
 
-def test_autodiff_accuracy():
-    pytest.importorskip("jax")
-    assert jnp is not None
-
-    sol = ct.Solution("mechs/sandiego.yaml", "gas")
-    ptk_base_cls = pyro.codegen.python.get_thermochem_class(sol)
-
-    ptk = make_jax_pyro_class(ptk_base_cls, jnp)
+    sol, ptk = pyro_init("sandiego", jnp, request)
 
     # mass ratios
     equiv_ratio = 1.0
     ox_di_ratio = 0.21
     stoich_ratio = 0.5
     # indices
-    i_fu = ptk.species_index("H2")
-    i_ox = ptk.species_index("O2")
-    i_di = ptk.species_index("N2")
+    i_fu = ptk.get_species_index("H2")
```

Comment:
`get_` or no? I'm on the fence. I think Cantera uses simply `species_index`, but then again, it doesn't use `get_` anywhere.

Reply:
See my other comment on `_get`.